pkg/operatorsource/operatorsource.go

```diff
@@ -0,0 +1,11 @@
+package operatorsource
+
+// OpsrcOwnerNameLabel is the label used to mark ownership over resources
+// that are owned by the OperatorSource. When this label is set, the reconciler
+// should handle these resources when the OperatorSource is deleted.
+const OpsrcOwnerNameLabel string = "opsrc-owner-name"
+
+// OpsrcOwnerNamespaceLabel is the label used to mark ownership over resources
+// that are owned by the OperatorSource. When this label is set, the reconciler
+// should handle these resources when the OperatorSource is deleted.
+const OpsrcOwnerNamespaceLabel string = "opsrc-owner-namespace"
```

Comment:
Are OpsrcOwnerNameLabel and OpsrcOwnerNamespaceLabel still being used anywhere?

Reply:
only in the tests afaik

Reply:
Then there is an issue that is not being uncovered. There should be no constants in the main code base that is only being used by the test code.

Reply:
+1

Reply:
Agreed. Do we want to remove the part of the test that checks for these labels? Or do we want to just move these constants to the testing package?

Reply:
We definitely don't want to move these constants to the testing package. That would just be moving the problem. You need to figure out what these constants were used for in the first place and how you have modified or maintaining that behavior.

Reply:
Because the catalogsourceconfig and operatorsource both delete the same child resources, the oprscOwnerLabel is no longer used to find child resources during deletion. These labels were used in an older version of the tests where we cared about this particular label. Now this test just checks that any label will be correctly propagated. During upgrade, because we want to delete the orphaned catalogsourceconfigs, we will need that label to identify which catalogsourceconfigs to delete so I will leave it in the builders package (this file `operatorsource.go` was created by mistake)